tools/labs/templates/networking/1-2-netfilter/user/test.c

```diff
@@ -11,6 +11,7 @@
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <sys/ioctl.h>
+#include <sys/systypes.h>
 #include <fcntl.h>
 #include <unistd.h>
```

Comment:
So, how does this work? Commit message says that you need to directly include ``<sys/sysmacros.h>`` but the code actually includes ``<sys/systypes.h>``.

Reply:
The commit message is correct and contents is wrong, my bad.

Reply:
Fixed now.

Reply:
Thanks Paul!